runtimes/pythonrt/worker_grpc_handler.go

```diff
@@ -279,7 +279,7 @@ func (s *workerGRPCHandler) StartSession(ctx context.Context, req *userCode.Star
 		return &userCode.StartSessionResponse{Error: err.Error()}, nil
 	}
 
-	return &userCode.StartSessionResponse{SessionId: resp.value.(sdktypes.SessionID).String()}, nil
+	return &userCode.StartSessionResponse{SessionId: resp.value.(string)}, nil
```

Comment:
`value`'s type is `any`, you can't just assume it's a string withtout checking the type first.

Reply:
Agree, the type assertion can panic.